spec/requests/survey_spec.rb

```diff
@@ -1,7 +1,75 @@
 require 'rails_helper'
 
-RSpec.describe 'Surveys', type: :request do
-  describe 'GET /admin/surveys' do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe 'Surveys' do
```

Comment:
instead of `Surveys` it could be `admin/surveys` and next level `GET #index`, `POST #index` instead of `GET admin/survey#index` etc.

Reply:
ordered a little bit. please see latest surveys tests